recipes/boltons/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "16.2.1" %}
+
+package:
+    name: boltons
+    version: '{{ version }}'
+
+source:
+    git_url: https://github.com/mahmoud/boltons.git
+    git_rev: '{{ version }}'
+
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+    run:
+        - python
+
+test:
+    requires:
+        - pytest
+    commands:
+        - python -c "from platform import system as s; import sys; sys.exit(int(s()!='Windows'))" || py.test --doctest-modules $SRC_DIR/boltons $SRC_DIR/tests
+        - python -c "from platform import system as s; import sys; sys.exit(int(s()=='Windows'))" || py.test --doctest-modules %SRC_DIR%/boltons %SRC_DIR%/tests
+
+about:
+    home: https://github.com/mahmoud/boltons
+    license: BSD
```

Comment:
@kalefranz boltons seems to be `BSD 3-Clause` can you confirm that and update it here?